pkg/nsx/services/securitypolicy/builder.go

```diff
@@ -1710,7 +1712,7 @@ func (service *SecurityPolicyService) updatePeerExpressions(obj *v1alpha1.Securi
 }
 
 func (service *SecurityPolicyService) buildShareName(nsxResourceID, groupName string) string {
-	nsxShareName := util.GenerateTruncName(common.MaxNameLength, fmt.Sprintf("%s-group-%s", nsxResourceID, groupName), "", common.ShareSuffix, "", "")
+	nsxShareName := util.GenerateTruncName(common.MaxNameLength, fmt.Sprintf("%s_group_%s", nsxResourceID, groupName), "", common.ShareSuffix, "", "")
```

Comment:
Is it better if we publish function `connectStrings`, and call it here other places where we manually use connect string like `fmt.Sprintf(%s_%s)`? And we can define a constant/variable for the default connector which doesn't need to expose.

Reply:
Good suggestion!  We expose ConnectStrings as a common function for connecting the strings. 
Also, we define a var for common usage.
`ConnectorUnderline string = "_"`